pyOCD/interface/hidapi_backend.py

```diff
@@ -107,3 +107,7 @@ def close(self):
         """
         logging.debug("closing interface")
         self.device.close()
+
+    def setPacketCount(self, count):
+        # No interface level restrictions on count
+        self.packet_count = count
```

Comment:
Actually, it had a pretty big impact, not quite doubling the speed for actual memory access.
<pre>
Target      Write Speed     Read Speed      USB speed       USB overlap speed

without     24.929752 KB/s  25.037943 KB/s  31.970114 KB/s  31.939022 KB/s  
with        44.660883 KB/s  45.513174 KB/s  31.969902 KB/s  63.973134 KB/s  
</pre>

This is on OS X. The hidapi implementation on the Mac uses a reader thread internally. For Windows it may be very different.